meson.build

```diff
@@ -169,6 +169,21 @@ elif get_option('enable-openblas')
 
 endif
 
+if get_option('enable-librom')
+   
+   assert(get_option('librom_root')!='',
+       'Must specify librom folder (-Dlibrom_root=path/to/libROM)')
```

Comment:
What is their default installation folder?

Reply:
What I'm looking for here is where the user decided to install libROM. The default location of the shared library is in libROM_install_directory/build/

Reply:
There is not a pkg-config file, but that's something I could request from the libROM folks

Reply:
Got it, even if they add pkg-config support it may be good to have this fallback.